django/chat/templates/chat/components/chat_security_label.html

```diff
@@ -1,7 +1,9 @@
 
 {% load i18n %}
 
-<div class="dropdown">
+<div class="dropdown"
+     id="chat-security-label-{{ chat.id }}"
+     {% if swap_oob %}hx-swap-oob="true"{% endif %}>
```

Comment:
The ID and hx-swap were needed for security label to update when a Folder with security > chat security is queried in RAG mode (it's not a very important feature, but might as well fix it).